server/athenian/api/become.py

```diff
@@ -11,7 +11,7 @@ def main():
     bearer = sys.argv[1]
     user = sys.argv[2] if len(sys.argv) > 2 else ""
     if not user.startswith("github|") and user:
-        with urlopen("https://api.github.com/users/" + user) as http:
+        with urlopen("https://api.github.com/user/" + user) as http:
```

Comment:
I wonder how it worked (and it did work) before...

Reply:
Actually, this does not work.

```
https://api.github.com/user/eiso
```

shows an error. This code is needed to resolve users by their logins.

Reply:
I think you probably misused the script. It has two modes:
1. `become.py <token> eiso` by gh login
2. `become.py <token> 'github|12345678'` by gh id

Reply:
ah indeed just read now 👍